src/utils/time_tracker.py

```diff
@@ -0,0 +1,12 @@
+from galaxyutils.time_tracker import TimeTracker as OGTimeTracker
+
+
+class TimeTracker(OGTimeTracker):
+    def get_tracking_games(self):
+        return [game_info.game_id for game_info in self._running_games_dict]
```

Comment:
I got an error for minecraft dungeons when tried to play for the first time:
AttributeError: 'str' object has no attribute 'game_id'

Reply:
Yeah, I actually forgot to test that. Sorry, completely my fault. I'll fix that ASAP. Thanks for finding that issue.

Reply:
@FriendsOfGalaxy, I've just fixed it. I'd appreciate it if you could test it with Minecraft Dungeons or just Minecraft as I don't have the ability to test right now.

Reply:
We might as well check it on the repo -> fog pull mr